manifests/kots-config.yaml

```diff
@@ -19,9 +19,20 @@ spec:
             If you need further info, please check [docs.carto.com](https://docs.carto.com).
 
             The ID of your Self-Hosted installation is: `{{repl LicenseFieldValue "selfHostedId" }}`
+
+        - name: backupReminder
+          type: label
+          title: |-
+            ⚠️  Don't forget to follow the CARTO docs regarding backups  ⚠️
+            You need to implement a backup policy for the next things in order to recover your data in case of upgrade, migration or disaster recovery:
+            ➡️ Database
+            ➡️ Config. ⚠️ We have some hidden configurations so please, follow the guide.
```

Comment:
We can review this text in the future, as it would be great to add a link to the documentation